test/common/common/hex_fuzz_test.cc

```diff
@@ -0,0 +1,14 @@
+#include "common/common/hex.h"
+
+#include "test/fuzz/fuzz_runner.h"
+
+namespace Envoy {
+namespace Fuzz {
+
+DEFINE_FUZZER(const uint8_t* buf, size_t len) {
+  Envoy::Hex::encode(buf, len);
```

Comment:
A hex encode/decode would work for a round-trip validation.

Reply:
Done